packages/flutter/test/material/icon_button_test.dart

```diff
@@ -337,6 +337,40 @@ void main() {
     await gesture.up();
   });
 
+  testWidgets('IconButton with explicit min splash radius',
+      (WidgetTester tester) async {
+    const double minSplashRadius = 10.0;
+
+    Widget buttonWidget = wrap(
+      child: IconButton(
+        icon: const Icon(Icons.android),
+        minSplashRadius: minSplashRadius,
+        onPressed: () {/* enable the button */},
```

Comment:
Comments should be structured like a sentence. i.e.
```suggestion
        onPressed: () {/* Enable the button. */},
```